superset-frontend/src/SqlLab/components/TableElement.jsx

```diff
@@ -209,15 +215,15 @@ class TableElement extends React.PureComponent {
           title={table.name}
           trigger={['hover']}
         >
-          <a
-            href="#"
+          <StyledSpan
+            dataTest="collapse"
```

Comment:
I think this should be a data-test attribute, for consistency. I'll add a suggestion for the test change too.
```suggestion
            data-test="collapse"
```